### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by gaaaork** [necanthrope/night-witches-pbta-foundryvtt#4]
Hi Necanthrope,

I hope you’re doing well. I’m reaching out because I’ve been trying to set up your "Night Witches" module for Foundry VTT and ran into some trouble. I'm really excited to try it out in my games!

Here's what I've done so far:

Downloaded the Module: I grabbed the ZIP file from your [GitHub repository](https://github.com/necanthrope/night-witches-pbta-foundryvtt).

Extracted the Files: I unzipped the files on my Mac.

Moved the Module Folder: I placed the extracted folder into the Foundry VTT modules directory on my system at /Users/[MyName]/Library/Application Support/FoundryVTT/Data/modules.

Activated the Module: I launched Foundry VTT and opened my world, but I couldn't find "Night Witches" in the module list, or it doesn’t activate properly.

Could you please let me know if there’s anything I might be missing or need to do differently? Any help you could offer would be amazing.

Thanks a lot for your hard work on this module! I’m looking forward to diving into the game with it.

Best,

gaaaork

**Comment by necanthrope** [comment]
Hi!  Thanks for reaching out!

The NW module is still in dvelopment, but you're welcome to use it.

It's dependent on the PBTA system being installed. Do you have that installed on your system?

**Comment by gaaaork** [comment]
Yes.  It's Installed on my system.  I just can't add the playbook ( Raven, ... ) to the players

**Comment by necanthrope** [comment]
Hey there!  I just made a new release. I've been working on other projects, hence the delay. If you choose to try it, please let me know if it works. 

https://github.com/necanthrope/night-witches-pbta-foundryvtt/releases/latest
